src/test/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerModificationTest.java

```diff
@@ -576,6 +577,215 @@ public void testPhaseTapChangerModification() throws Exception {
 
     }
 
+    private TwoWindingsTransformer createPhaseTapChanger() {
+        return createPhaseTapChanger(PhaseTapChanger.RegulationMode.FIXED_TAP);
+    }
+
+    private TwoWindingsTransformer createPhaseTapChanger(PhaseTapChanger.RegulationMode regulationMode) {
+        TwoWindingsTransformer twt3 = createTwoWindingsTransformer(getNetwork().getSubstation("s1"), "trf3", "trf3", 2.0, 14.745, 0.0, 3.2E-5, 400.0, 225.0,
+            41, 151, getNetwork().getVoltageLevel("v1").getId(), getNetwork().getVoltageLevel("v2").getId(),
+            "trf3", 1, ConnectablePosition.Direction.TOP,
+            "trf3", 2, ConnectablePosition.Direction.TOP);
+        Terminal phaseTapChangerTerminal = ModificationUtils.getInstance().getTerminalFromIdentifiable(getNetwork(),
+            "v3load",
+            "LOAD",
+            "V3");
+        twt3.newPhaseTapChanger()
+            .setLowTapPosition(0)
+            .setTapPosition(1)
+            .setRegulationTerminal(phaseTapChangerTerminal)
+            .setRegulationMode(regulationMode)
+            .beginStep()
+            .setR(39.78473)
+            .setX(39.784725)
+            .setG(0.0)
+            .setB(0.0)
+            .setRho(1.0)
+            .setAlpha(1.)
+            .endStep()
+            .beginStep()
+            .setR(39.78475)
+            .setX(39.784727)
+            .setG(0.0)
+            .setB(0.0)
+            .setRho(1.0)
+            .setAlpha(1.1)
+            .endStep()
+            .add();
+        return twt3;
+    }
+
+    @Test
+    public void testPhaseTapChangerRegulationModification() throws Exception {
+        TwoWindingsTransformer twt3 = createPhaseTapChanger();
+        String twtId = "trf3";
+        // modification 1 : FIXED_TAP -> CURRENT_LIMITER
+        TwoWindingsTransformerModificationInfos phaseTapChangerCreation = TwoWindingsTransformerModificationInfos.builder()
+            .stashed(false)
+            .equipmentId(twtId)
+            .ratioTapChanger(RatioTapChangerModificationInfos.builder()
+                .build())
+            .phaseTapChanger(PhaseTapChangerModificationInfos.builder()
+                .enabled(new AttributeModification<>(true, OperationType.SET))
+                .regulationMode(new AttributeModification<>(PhaseTapChanger.RegulationMode.CURRENT_LIMITER, OperationType.SET))
+                .regulationValue(new AttributeModification<>(10.0, OperationType.SET))
+                .build())
+            .build();
+
+        String modificationToModifyJson = mapper.writeValueAsString(phaseTapChangerCreation);
+        mockMvc.perform(post(getNetworkModificationUri()).content(modificationToModifyJson).contentType(MediaType.APPLICATION_JSON))
+            .andExpect(status().isOk()).andReturn();
```

Comment:
```suggestion
            .andExpect(status().isOk());
```
result is not used, do not need .andReturn();